app/views/support_interface/validation_errors/_list.html.erb

```diff
@@ -0,0 +1,47 @@
+<table class="govuk-table">
```

Comment:
move partials back into respective folders as they turned out to be quite different